### ISSUE TITLE
net-p2p/gridcoin-5.0.0.0 fails to build due to boost build errors

### ISSUE THREAD

**Issue by NuLL3rr0r** [nethershaw/gridcoin#7]
This is my installed boost version:

```
[ebuild   R    ] dev-libs/boost-1.74.0:0/1.74.0::gentoo  USE="bzip2 icu nls python threads zlib -context -debug -doc -lzma -mpi -numpy -static-libs -tools -zstd" ABI_X86="(64) -32 (-x32)" PYTHON_TARGETS="python2_7 python3_7 -python3_6 -python3_8 -python3_9" 0 KiB
```

[build.log](https://github.com/nethershaw/gridcoin/files/5211811/build.log)

**Comment by commander-derp** [comment]
Same problem, downgrading boost to 1.72 worked.
My recommendation for the ebuild is set the RDEPEND to <boost-1.74 until the gridcoin source is updated to work properly with newer versions.

Didn't test with 1.73 as it's not in my portage anymore.

**Comment by NuLL3rr0r** [comment]
Thank you so much. It's great to hear it has been solved in 5.0.1. I'm going to try it.
